lib/broccoli/ember-app.js

```diff
@@ -1534,6 +1534,8 @@ EmberApp.prototype.toTree = function(additionalTrees) {
  */
 EmberApp.prototype.contentFor = function(config, match, type) {
   var content = [];
+  var deprecatedHooks = ['app-prefix', 'app-suffix'];
+  var deprecate = this.project.ui.writeDeprecateLine.bind(this.project.ui);
```

Comment:
@gmurphey @rwjblue Fun story, the API for `writeDeprecateLine` is reversed from the API for `deprecate`/`deprecateUI`. Threw me for a loop.